pkg/gitparse/gitparse.go

```diff
@@ -209,7 +214,14 @@ func NewParser(options ...Option) *Parser {
 // RepoPath parses the output of the `git log` command for the `source` path.
 // The Diff chan will return diffs in the order they are parsed from the log.
 func (c *Parser) RepoPath(ctx context.Context, source string, head string, abbreviatedLog bool, excludedGlobs []string, isBare bool) (chan *Diff, error) {
-	args := []string{"-C", source, "log", "-p", "--full-history", "--date=format:%a %b %d %H:%M:%S %Y %z"}
+	args := []string{
+		"-C", source,
+		"log",
+		"--patch", // https://git-scm.com/docs/git-log#Documentation/git-log.txt---patch
+		"--full-history",
+		"--date=format:%a %b %d %H:%M:%S %Y %z",
+		"--pretty=fuller", // https://git-scm.com/docs/git-log#_pretty_formats
```

Comment:
Are we worried that the pre-canned pretty formats might change in the future?

Reply:
I'm not worried. The formats seem to be consistent. 
https://web.archive.org/web/20140208065758/https://git-scm.com/docs/git-log